src/go/util/transport_socket_test.go

```diff
@@ -152,14 +155,18 @@ func TestCreateDownstreamTransportSocket(t *testing.T) {
 									"filename":"/etc/ssl/endpoints/server.key"
 								}
 							}
-						]
+						],
+						"tlsParams":{
+							"tlsMinimumProtocolVersion":"TLSv1_1"
+						}
 					}
 				}
 			} `,
 		},
 		{
-			desc:    "Downstream Transport Socket for TLS, for legacy ESPv1",
-			sslPath: "/etc/nginx/ssl",
+			desc:               "Downstream Transport Socket for TLS, for legacy ESPv1",
+			sslPath:            "/etc/nginx/ssl",
+			sslMaximumProtocol: "TLSv1.3",
```

Comment:
can we add a test to have both min and max?

Reply:
Done